src/assets/ListedDerivative/CommodityFuture/commodityFuture.test.js

```diff
@@ -0,0 +1,57 @@
+import CommodityFuture from './commodityFuture'
+import { Comment, Reference } from '../../../children'
+import AssetLink from '../../../children/Link/AssetLink'
+
+describe('CommodityFuture', () => {
+    describe('constructor', () => {
+
+        it('should set default issueDate correctly', () => {
+          const testCommodityFuture = new CommodityFuture({})
+          const issueDate = testCommodityFuture.issueDate
+          expect(issueDate).toEqual('0001-01-01')
+        })
+
+        it('should set links to empty object if no Links', () => {
+            const testCommodityFuture = new CommodityFuture({})
+            expect(testCommodityFuture.links).toEqual({})
+        })
+
+        it('should set links as an object of <string, Link> type', () => {
+            const links = {
+                'Single': [{
+                    linkedAssetId: 'testId1',
+                    active: true
+                }],
+                'Multiple': [{
+                    linkedAssetId: 'testId2',
+                    active: true
+                }, {
+                    linkedAssetId: 'testId3',
+                    active: false
+                }]
+            }
+            const testCommodityFuture = new CommodityFuture({ links })
+            expect(testCommodityFuture.links.Single[0]).toBeInstanceOf(AssetLink)
+        })
+
+        it('should set references correctly (inclduing the AMaaS reference)', () => {
```

Comment:
The AMaaS reference is no longer set here, we will let the back end do it. You don't have to change the test, just remove the line about including the AMaaS reference.